symbols.yaml

```diff
@@ -63,7 +63,7 @@ punctuator:
     '@' : [ '@', ©, ®, ℗ ]
     '#' : [ '#', № ]
     '%' : [ '%', ％, '°', '℃', ‰, ‱, ℉, ℅, ℆, ℀, ℁, ⅍ ]
-    '$' : [ ￥, '$', '€', '£', '¥', '¢', '¤', ₩ ]
+    '$' : [ ￥, '$', '€', '£', '¥', '¢', '¤', ₩, ￦ ]
```

Comment:
Again, why add the full shape one under half_shape here?

Reply:
Because the full-width `￥` is already there. Just like `￥ `, full-width `￦` is preferred when used among CJK characters and numerals while still typing in half-shape mode.

Reply:
Ok, if full shape ￦ is used prevalently in Korean, it's fine